starter-code/client/src/app/app.module.ts

```diff
@@ -3,18 +3,32 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 
+
+
+import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
+import { JournalServicesService } from './services/journal-services.service';
+import { HomepageComponent } from './pages/homepage/homepage.component';
+import { SingleEntryComponent } from './components/single-entry/single-entry.component';
```

Comment:
Be clean! 
Separate the angular components from the components from the pages from the services from the guards.